spec/lib/ruby_smb/client_spec.rb

```diff
@@ -1015,12 +1015,6 @@
           bogus_response.smb_header.command = 0xff
           expect { smb1_client.negotiate_response(bogus_response.to_binary_s) }.to raise_error(RubySMB::Error::InvalidPacket)
         end
-
-        it 'considers the response invalid if Extended Security is not enabled' do
-          bogus_response = smb1_extended_response
-          bogus_response.parameter_block.capabilities.extended_security = 0
-          expect { smb1_client.negotiate_response(bogus_response.to_binary_s) }.to raise_error(RubySMB::Error::InvalidPacket)
-        end
```

Comment:
Instead of deleting the test, is it worth updating it to reflect the new implementation? 👀

Reply:
https://youtu.be/5r06heQ5HsI